README.md

```
@@ -1,40 +1,256 @@
+</div>
 
-First, run the development server:
+코드잇 미래내일 일경험 인턴에서 진행한 프로젝트입니다.
+
+<br/>
+
+# 💡 주요 업데이트 내용 (2.0)
+
+> 이전 버전([Codeit Resources 1.0](https://github.com/codeit-2000/codeit-resources))에서 다음과 같은 개선사항이 추가되었습니다.
+
+### 1. Google Calendar 연동
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)